### ISSUE TITLE
How to get Fig.4. (a) 0.85 ?

### ISSUE THREAD

**Issue by wangqiim** [chenbong/ARM-Net#4]
In Fig.4.(a), I find different models will cause different PSNR lookup results. How to get Spearman correlation (0.85) coefficient picture (Fig.4.(a)). What model do you use to generate psnr values. (complex model will cause more pricise PSNR value)

**Comment by mrgreen3325** [comment]
> In Fig.4.(a), I find different models will cause different PSNR lookup results. How to get Spearman correlation (0.85) coefficient picture (Fig.4.(a)). What model do you use to generate psnr values. (complex model will cause more pricise PSNR value)

Hello.
I find errors when run the test with fsrcnn with the pretrain model.
  sr_img[i*step*self.scale:i*step*self.scale+patch_size*self.scale,j*step*self.scale:j*step*self.scale+patch_size*self.scale,:]+=sr_list[index]
IndexError: list index out of range
It seems that it occur during calculate the test_patch in is_calc_hmap .
Do you have the similar problem?

**Comment by wangqiim** [comment]
> > In Fig.4.(a), I find different models will cause different PSNR lookup results. How to get Spearman correlation (0.85) coefficient picture (Fig.4.(a)). What model do you use to generate psnr values. (complex model will cause more pricise PSNR value)
> 
> Hello. I find errors when run the test with fsrcnn with the pretrain model. sr_img[i_step_self.scale:i_step_self.scale+patch_size_self.scale,j_step_self.scale:j_step_self.scale+patch_size_self.scale,:]+=sr_list[index] IndexError: list index out of range It seems that it occur during calculate the test_patch in is_calc_hmap . Do you have the similar problem?

No, I don't run the base code actually. I just modify and use some codes about psnr-lookup-table on my dataset.
